### PR TITLE
Add docs cta 2

### PR DESCRIPTION
**Pull request by brock-statsig**
Replaces https://github.com/statsig-io/docs/pull/1165 (Sorry Matt)

Includes a new CTA in the navbar with a link out to console, plus tracking the number of clicks on that on Statsig. 

Includes two new MVP tutorials on how to add feature flags or AB tests to a very simple Flask/Python project.

### CODE REVIEW

File: docs/guides/python-flask-abtests.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Python/Flask A/B Test
+title: Python/Flask A/B Test
```
Comment (matt-statsig) [inline docs/guides/python-flask-abtests.mdx:3]: can this be "How to set up feature flags in python using flask?" maybe? Posthogs is "[How to set up Python feature flags in Flask](https://posthog.com/tutorials/python-feature-flags)" which feels closer to what people would google

File: docs/guides/python-flask-abtests.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Next we'll start editing app.py - we'd recommend opening it up in your favourite code editor to get started. If you're new at this, you can open it in any text editor. We'll start with the simplest possible, 6-line flask app. 
+
+```Python 
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return f"Hello World!"
```
Comment (matt-statsig) [inline docs/guides/python-flask-abtests.mdx:36]: I think if you're not referencing any variables in the string with {variable} we should not use the f

File: docs/guides/python-flask-abtests.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+![image](/img/red_backgroundcolor.png)
+
+What we'd like to do is make our app show a different colour for each user using an A/B test (what we call an "experiment") in Statsig. If our app had more functionality, we could then see which colour users preferred or which supported our business metrics, and made a decision to choose that option. Even without building more functionality, Statsig could show you differences in metrics like WAU and DAU with a single line of code!.
+
+Lets set up an Experiment in Statsig!
+
+## Step 3: Setting Up Statsig
+
+If you haven't yet - now is the time to head to [console.statsig.com](http://console.statsig.com) and set up an account - Statsig is free up to 1 Million events per month for A/B Tests. When you signup, you'll have the opportunbity to setup your SDK type, but if you don't see this, no problem, simply click on the settings gear in your home page then click on Keys & Environments, where you can copy your server key. Initializing Statsig in Python and Flask only takes two lines:
```
Comment (matt-statsig) [inline docs/guides/python-flask-abtests.mdx:67]: opportunbity

File: docs/guides/python-flask-feature-flags.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: Python/Flask Feature Flags
+title: Python/Flask Feature Flags
```
Comment (matt-statsig) [inline docs/guides/python-flask-feature-flags.mdx:3]: same title comment as above

File: docs/guides/python-flask-feature-flags.mdx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    if(statsig.check_gate(user, "secret_route")):
+        visit_count += 1
+        return f"This page has been visited {visit_count} times."
+    else:
+        return "failed feature flag"
+```
+
+Now, we've added an email field to our user - which we can pass in in a query parameter, for example - http://127.0.0.1:5000/secretRoute?email=brock@statsig.com. Now, lets go change our rule in the the Statsig console (pressing the back button) to show only users where their email contains @statsig.com - and pass 100% of those users.
+
+![iamge](/img/statsigemail_gate.png)
```
Comment (matt-statsig) [inline docs/guides/python-flask-feature-flags.mdx:120]: is iamge a typo?

File: docusaurus.config.js [hunk 6]
```diff
@@ -45,7 +45,13 @@ module.exports = {
         srcDark: "img/logo_white.svg",
         href: "https://statsig.com",
       },
-      items: [],
+      items: [
+        {
+          type: 'html',
+          position: 'right',
+          value: '<button id="consoleCTA" onclick="window.open(\'https://console.statsig.com\', \'_blank\').focus(); window.statsig.logEvent({}, \'console_button_click\', window.location.pathname, {referrer: document && document.referrer,});">Get Started</button>',
```
Comment (matt-statsig) [inline docusaurus.config.js:52]: not for this one, let's submit, but we should figure out if this should pass a GA token. Also is referrer: document enough for us to know this came from docs? just curious what type of string is appended here and if we can filter for all docs easily

**Review by matt-statsig — APPROVED** [review]
nice!!

File: docs/guides/python-flask-abtests.mdx (hunk 2 above)
Comment (brock-statsig) [reply docs/guides/python-flask-abtests.mdx:36]: Thank you, actioned!

File: docs/guides/python-flask-abtests.mdx (hunk 1 above)
Comment (brock-statsig) [reply docs/guides/python-flask-abtests.mdx:3]: Done! Thank you.

File: docs/guides/python-flask-feature-flags.mdx (hunk 5 above)
Comment (brock-statsig) [reply docs/guides/python-flask-feature-flags.mdx:120]: Apparently still works! But yes

File: docusaurus.config.js (hunk 6 above)
Comment (brock-statsig) [reply docusaurus.config.js:52]: I copied this code from link tracking in docs, confirmed it works but I actually haven't seen the referrer be populated yet. Definitely some TODO here

**Pull request merged**